none.yaml

```diff
@@ -151,35 +157,53 @@ resources:
             else
               echo "sources.list was not updated based on bootImage"
             fi
-
             # Telegraf agent install
             if [ $bootImage = "Ubuntu1604" ] || [ $bootImage = "Ubuntu1804" ]
             then
               curl -L -O https://dl.influxdata.com/telegraf/releases/telegraf_1.10.3-1_amd64.deb
               dpkg -i telegraf_1.10.3-1_amd64.deb
+              apt-get update -y
```

Comment:
avoid doing `apt-get update` in Heat Stacks, you'll have to change this in every stack.